pyspark/pysparkbuilddc.json

```diff
@@ -217,11 +223,26 @@
                         ],
                         "resources": {},
                         "terminationMessagePath": "/dev/termination-log",
-                        "imagePullPolicy": "Always"
+                        "imagePullPolicy": "Always",
+                        "volumeMounts": [
+                           {
+                              "mountPath": "/etc/oshinko-spark-configs",
```

Comment:
ditto the path, it can be set in an environment variable (but not queried through a parameter, it will still be a fixed value)